src/Compilers/CSharp/Portable/Symbols/Source/LocalFunctionSymbol.cs

```diff
@@ -36,14 +35,18 @@ internal sealed class LocalFunctionSymbol : MethodSymbol
             Binder binder,
             Symbol containingSymbol,
             LocalFunctionStatementSyntax syntax)
+            : base(containingSymbol, syntax.GetReference(), syntax.ExpressionBody?.GetReference() ?? syntax.Body?.GetReference(), syntax.Location)
```

Comment:
>syntax.ExpressionBody?.GetReference() ?? syntax.Body?.GetReference() [](start = 60, length = 68)

This looks inconsistent with other functions where Body is preferred over ExpressionBody.